public/regdesk.html

```diff
@@ -53,13 +53,19 @@
         </div>
       </div>
     </nav>
-    <div class="row container-no-gutter">
+    <div class="row row-no-margin container-no-gutter">
       <div id="buttonGutterLeft" class="col-2 d-grid gap-2 container-no-gutter">
         <button id="cancelButton" class="btn btn-danger btn-fullheight">CANCEL</button>
       </div>
       <div class="col-8 pt-3">
-        <div class="d-grid gap-2 mb-3">
-          <button class="btn btn-primary" id="newSearch">🔎 New Search</button>
+        <div class="row">
+          <div class="col-4 offset-4 d-grid gap-2 mb-3">
+            <button class="btn btn-primary" id="newSearch">🔎 New Search</button>
+          </div>
+        </div>
+        <div id="searchResults" class="row">
+          <!-- Demonstration canvas, actual layout will be different. -->
+          <canvas id="canvas" width="589" height="193" style="background-color: white; width: 589px; height: 193px; padding: 0;"></canvas>
```

Comment:
Yes because something somewhere in Bootstrap was messing with it. Setting the value with either CSS or with the HTML values was insufficient, both were necessary in my (admittedly insufficient) testing. I planned to narrow down exactly what was going on there when I built out the proper interface instead of just this testing element.